sensehat/src/main/java/com/google/android/things/contrib/driver/sensehat/BaroTemp.java

```diff
@@ -0,0 +1,190 @@
+    private final int LPS_CTRL_REG3 = 0x22; //  Interrupt configuration
+    private final int LPS_CTRL_REG4 = 0x23; //  Interrupt configuration
+
+    private final int LPS_INTERRUPT_CFG = 0x24;//  Interrupt configuration
+    private final int LPS_INT_SOURCE = 0x25; //  Interrupt source configuration
+
+    private final int LPS_STATUS_REG = 0x27; //  Status (new pressure/temperature data available)
+
+    private final int LPS_PRESS_OUT_XL = 0x28; //  Pressure output, loweste byte
+    private final int LPS_PRESS_OUT_L = 0x29; //  Pressure output, low byte
+    private final int LPS_PRESS_OUT_H = 0x2A; //  Pressure output, high byte
+
+    private final int LPS_TEMP_OUT_L = 0x2B; //  Temperature output, low byte
+    private final int LPS_TEMP_OUT_H = 0x2C; //  Temperature output, high byte
+
+    private final int LPS_FIFO_CTRL = 0x2E; //  FIFO control / mode selection
+    private final int LPS_FIFO_STATUS = 0x2F; //  FIFO status
+
+    private final int LPS_THS_P_L = 0x30; //  Pressure interrupt threshold, low byte
+    private final int LPS_THS_P_H = 0x31; //  Pressure interrupt threshold, high byte
+
+    // The next two registers need special soldering
+    private final int LPS_RPDS_L = 0x39;//  Pressure offset for differential pressure computing, low byte
+    private final int LPS_RPDS_H = 0x3A; //  Differential offset, high byte
+    private int mMillibarAdjust = 0;
+
+    /**
+     * Create a new barometric pressure and temperature sensor driver connected on the given I2C bus.
+     *
+     * @param bus I2C bus the sensor is connected to.
+     * @throws IOException when a lower level does
+     */
+    public BaroTemp(String bus) throws IOException {
+        PeripheralManager pioService = PeripheralManager.getInstance();
+        mDevice = pioService.openI2cDevice(bus, SenseHat.I2C_LPS25H_ADDRESS);
+        // power down first
+        mDevice.writeRegByte(LPS_CTRL_REG1, (byte) 0);
+        // power up, data rate 12.5Hz (10110000)
+        mDevice.writeRegByte(LPS_CTRL_REG1, (byte) 0xb0);
+        if (0xBD != (mDevice.readRegByte(LPS_WHO_AM_I) & 0xFF)) {
+            throw new IOException("This does not seem to be a LPS25H");
+        }
+    }
+
+    @Override
+    public void close() throws IOException {
+        if (mDevice != null) {
+            try {
+                // power down device
+                mDevice.writeRegByte(LPS_CTRL_REG1, (byte) 0);
+            } catch (Exception any) {
+                // we tried
+            }
+            try {
+                mDevice.close();
+            } finally {
+                mDevice = null;
+            }
+        }
+    }
+
+    private int readSigned24(int a0, int a1, int a2) throws IOException {
+        int ret = (mDevice.readRegByte(a0) & 0xFF);
+        ret += ((int) mDevice.readRegByte(a1) & 0xFF) << 8;
+        ret += ((int) mDevice.readRegByte(a2) & 0xFF) << 16;
+        if (ret < 8388608) return ret;
```

Comment:
[Google's style guide](https://google.github.io/styleguide/javaguide.html#s4.1-braces) states that conditionals should be wrapped in curly braces, even if it's one line,

ie. 
```java
if (ret < 8388608) {
    return ret;
}
```